supplementary_style_guide/glossary_terms_conventions/general_conventions/g.adoc

```diff
@@ -201,7 +201,7 @@
 [discrete]
 [[gnome-classic]]
 ==== image:images/yes.png[yes] GNOME Classic (noun)
-*Description*: Although the desktop team tends to refer to "GNOME Classic" (technically, GNOME Shell with the classic mode extensions enabled) as "classic mode" in internal and developer-oriented community documents, we should stay consistent with what is exposed to the user on the GNOME Display Manager (GDM) login screen, that is, "GNOME Classic". The GNOME "modern mode" (technically, GNOME Shell with the classic mode extensions disabled) is referred to as "GNOME" (on the login screen and elsewhere).
+*Description*: Although the desktop team tends to refer to _GNOME Classic_ (technically, GNOME Shell with the classic mode extensions enabled) as _classic mode_ in internal and developer-oriented community documents, we should stay consistent with what is displayed to the user on the GNOME Display Manager (GDM) login screen, that is, "GNOME Classic". Refer to GNOME _modern mode_ (technically, GNOME Shell with the classic mode extensions disabled) as "GNOME" (on the login screen and elsewhere).
```

Comment:
_GNOME Classic_ > "GNOME Classic" because this sentence discusses how the term is written? Same with "GNOME modern mode"?

Reply:
Fixed per your suggestions.